code/game/machinery/machinery.dm

```diff
@@ -540,8 +540,11 @@ Class Procs:
  * @param user /mob The mob that used the emag.
  */
 /obj/machinery/proc/emag(mob/user as mob)
-	// Disable emaggability.
+	// Disable emaggability. Note that some machines such as the Communications Computer might be emaggable multiple times.

```

Comment:
Well I personally think that disabling emaggability on the parent proc is stupid, that proc should be empty. (save for the spark and sound thing)